sale_order_margin_percent/models/sale_order_margin.py

```diff
@@ -17,3 +17,19 @@ def _compute_percent(self):
         for order in self:
             if order.margin and order.amount_untaxed:
                 order.percent = (order.margin / order.amount_untaxed) * 100
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    percent = fields.Float(
+        string='Percent',
+        compute='_compute_percent',
+        digits=(16, 2),
+        )
```

Comment:
```
    margin_percent = fields.Float(
        string='Margin (%)',
        compute='_compute_percent',
        digits=dp.get_precision('Product Price'),
        )
```

to be consistent with accout_invoice_margin module. (same repository)
https://github.com/OCA/margin-analysis/blob/12.0/account_invoice_margin/models/account_invoice.py#L95